community: Add bio for Razvan Nitu (#6)
open-education-hub/open-education-hub-site: website/docs/community/people.md

# Project Team

## Răzvan Nițu

As part of the OpenEdu (21-COP-0016) team I am charged with defining and maintaining the methodology for creating high quality educational resources.

> I am a PhD student at university POLITEHNICA of Bucharest in the field of computer science and engineering.

Reviewer: ```suggestion
I am a PhD student at University POLITEHNICA of Bucharest in the field of computer science and engineering.
```

Reviewer 2: s/in the field of computer science and engineering/in Computer Science and Engineering/

My thesis aims at optimiziming memory operations at all possible levels: application, operating system and hardware.

> I am an active contributor to an open source programming language called [D](https://dlang.org/).

Reviewer 2: ```suggestion
I am an active open source contributor to [the D programming language](https://dlang.org/), a modern programming language focusing on security and ... TODO.
```

I currently work at [Bitdefender](https://www.bitdefender.com/) as a software engineer, part of the [Connected Home Security](https://www.bitdefender.com/smart-home/) team.

> Apart from computer science, I really enjoy skiing, dancing and cooking.

Author: I don't know what happened here.

Reviewer: copycat

Reviewer 2: What is this?

Author: My guess is Liza's sentence finished with an EOF and I added a newline.

## Răzvan Nițu

> ![Razvan Nitu](../../static/img/people/liza_babu.jpeg)

Reviewer 2: This is a link to Liza's picture. Fix it with a link to your picture.

As part of the OpenEdu (21-COP-0016) team I am charged with defining and maintaining the methodology for creating high quality educational resources.

> I am a PhD student at University POLITEHNICA of Bucharest in Computer Ccience and Engineering.

Reviewer: ```suggestion
I am a PhD student at University POLITEHNICA of Bucharest in Computer Science and Engineering.
```
